frontend/src/shared/components/Toast/store/toastActions.ts

```diff
@@ -0,0 +1,103 @@
+import { DEFAULT_DURATION_MS, DEFAULT_TYPE } from "../constants/toast.constants";
+import type { ToastData, ToastOptions, ToastsState } from "../types/toast.type";
+import { toastStore } from "./toastStore";
+
+const getDistributedToasts = (state: ToastsState) => {
+  const { toasts, defaultPosition, limit } = state;
+  const queue: ToastData[] = [];
```

Comment:
```suggestion
  const pedingQueue: ToastData[] = [];
```
대기 큐라는 것을 명시해도 좋을 것 같습니다~

Reply:
`pendingQueue`를 의도하신 거겠죠?